app/Nova/RemoteAttendanceLink.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable Generic.Strings.UnnecessaryStringConcat.Found
+
+namespace App\Nova;
+
+use App\Models\RemoteAttendanceLink as RAL;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use Laravel\Nova\Fields\DateTime;
+use Laravel\Nova\Fields\MorphTo;
+use Laravel\Nova\Fields\Text;
+use Laravel\Nova\Panel;
+
+/**
+ * A Nova resource for remote attendance links.
+ */
+class RemoteAttendanceLink extends Resource
+{
+    /**
+     * The model the resource corresponds to.
+     *
+     * @var string
+     */
+    public static $model = RAL::class;
+
+    /**
+     * The single value that should be used to represent the resource when being displayed.
+     *
+     * @var string
+     */
+    public static $title = 'id';
+
+    /**
+     * The logical group associated with the resource.
+     *
+     * @var string
+     */
+    public static $group = 'Meetings';
+
+    /**
+     * Indicates if the resource should be displayed in the sidebar.
+     *
+     * @var bool
+     */
+    public static $displayInNavigation = false;
+
+    /**
+     * The relationships that should be eager loaded on index queries.
+     *
+     * @var array<string>
+     */
+    public static $with = ['attendable'];
+
+    /**
+     * Indicates if the resource should be globally searchable.
+     *
+     * @var bool
+     */
+    public static $globallySearchable = false;
+
+    /**
+     * The list of recommended values for the note field.
+     *
+     * @var array<string>
+     *
+     * @phan-suppress PhanReadOnlyPublicProperty
+     */
+    public static $recommendedNotes = [
+        'Electrical Subteam', 'Mechanical/Mechatronics Subteam', 'Software Subteam', 'Whole Team', 'Training',
+        'Trainers Only', 'Discipline Core (n/a)', 'Event (n/a)',
```

Comment:
What's with the `(n/a)`?

Reply:
I think I originally called the field "Subteam" or something along those lines. I'll remove the `(n/a)`.